addons/web/static/src/core/dropdown/dropdown.js

```diff
@@ -336,5 +339,9 @@ Dropdown.props = {
         type: Object,
         optional: true,
     },
+    showCaret: {
+        type: Boolean,
+        optional: true,
```

Comment:
you can add a default value of false, and then `this.showCaret = this.props.showCaret || this.parentDropdown`